src/server/scriptInfo.ts

```diff
@@ -642,4 +644,13 @@ namespace ts.server {
             }
         }
     }
+
+    /*@internal*/

```

Comment:
Nit/FYI: I think `@internal` shouldn’t be needed since this isn’t exported.

Reply:
Thanks - removed.